lib/src/settings.rs

```diff
@@ -108,12 +108,6 @@ impl UserSettings {
             .unwrap_or(false)
     }
 
-    pub fn use_progress_indicator(&self) -> bool {
-        self.config
-            .get_bool("ui.progress-indicator")
-            .unwrap_or(true)
-    }
-
```

Comment:
Why is this moved out, and what does it have to do with the pager?

Reply:
See this PR's second commit. Since this is specific to the CLI, it shouldn't be in the lib crate.